File: tools/SepaTools/src/main/java/net/dankito/banking/sepa/camt/camt052_simplexml/Document.java
package net.dankito.banking.sepa.camt.camt052_simplexml;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;


@Root(name = "Document", strict = false)
public class Document {

    private BankToCustomerAccountReport bkToCstmrAcctRpt = null;

    public BankToCustomerAccountReport getBkToCstmrAcctRpt() {
        return bkToCstmrAcctRpt;
    }

    @Element(name = "bkToCstmrAcctRpt")
    public void setBkToCstmrAcctRpt(BankToCustomerAccountReport bkToCstmrAcctRpt) {
        this.bkToCstmrAcctRpt = bkToCstmrAcctRpt;
    }
}

File: tools/SepaTools/src/main/java/net/dankito/banking/sepa/camt/camt052_tikxml/Document.kt
package net.dankito.banking.sepa.camt.camt052_tikxml

import com.tickaroo.tikxml.annotation.Element
import com.tickaroo.tikxml.annotation.Xml


@Xml(name = "Document")
class Document {

    @Element(name = "BkToCstmrAcctRpt")
    var bkToCstmrAcctRpt: BankToCustomerAccountReport? = null

}